tests/examples.js

```diff
@@ -534,6 +578,7 @@ metasync.composition([
   concurrentQueueTest,
   concurrentQueuePauseResumeStopTest,
   throttleTest,
+  debaunceTest,
```

Comment:
It's not your fault but concurrentQueuePauseResumeStopTest is broken. Because of that your test doesn't even execute when `npm run test` (same as `throttleTest` and all next)